src/scripts/main.js

```diff
@@ -2,4 +2,41 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+
+  wall.addEventListener('click', () => {
+    const wallRect = wall.getClientRects();
+    const spiderRect = spider.getBoundingClientRect();
+
+    const spiderHalfWidth = spiderRect.width / 2;
+    const spiderHalfHeight = spiderRect.height / 2;
+    const borderWidth = 10;
+
+    const clickX = event.clientX - wallRect.left;
+    const clickY = event.clientY - wallRect.top;
```

Comment:
The `event` object is not defined in this scope. You should use the `e` parameter from the outer event listener function instead.